db/data_migrations/20200921120101_delete_05004_footnote.rb

```diff
@@ -0,0 +1,51 @@
+TradeTariffBackend::DataMigrator.migration do
+  name 'Delete 05044 footnote for 7304293080 commodity'
+
+  up do
+    applicable do
+      FootnoteAssociationGoodsNomenclature::Operation.where(
+        goods_nomenclature_sid: 86621,
+        footnote_id: '004',
+        footnote_type: '05',
+        goods_nomenclature_item_id: '7304293080',
+        operation: 'D'
```

Comment:
I would create a update operation to just endate the record

Reply:
fixed.

Reply:
Just got confirmation the end date was 16.05.2020